### ISSUE TITLE
Show "waiting/queued" for queued jobs

### ISSUE THREAD

**Issue by kallisti5** [concourse/concourse#5040]
If all your workers are busy, queued jobs show as "running" on the dashboard, but the logs show:
"All workers are busy at the moment, please stand-by."

It would be nice if the dashboard showed a status other than running for queued jobs.  Here's an example:

![Screen Shot 2020-01-17 at 3 52 52 PM](https://user-images.githubusercontent.com/100110/72648925-df494580-3941-11ea-8a11-fa57db5f74e3.png)

Several of those are waiting on a worker to be available... only one of them are actually building. Guess which one! :-)

**Comment by kallisti5** [comment]
Version is 5.7.1

**Comment by wagdav** [comment]
In February @aledeganopix4d  submitted an [RFC](https://github.com/concourse/rfcs/pull/43)  which would make this feature request possible to implement.

This ticket is blocked on that.

**Comment by taylorsilva** [comment]
# Issue Bankruptcy

This issue is being closed as part of 2025's **Issue Bankruptcy** declaration. [See here for details](https://blog.concourse-ci.org/posts/2025-01-06-concourse-2024-review/)

If you think this issue still has merit then feel free to open a new issue and reference this one.
